lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoTypeDeclarationImpl.kt

```diff
@@ -45,6 +45,56 @@ internal class KoTypeDeclarationImpl private constructor(
 
     override val isNullable: Boolean by lazy { ktTypeReference.text.last() == '?' }
 
+    override val isKotlinType: Boolean by lazy {
+        if (isImportAlias()) {
+            false
```

Comment:
I improve this code, because before it fails when for example type was named: `SampleCollection` (because contains `Collection`) or "ShortHair" (because contains `Short`).